source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -1126,6 +1127,300 @@ namespace AdaptiveCards { namespace Rendering { namespace Uwp
         THROW_IF_FAILED(textConfig->get_MaxWidth(&maxWidth));
         StyleXamlTextBlock(textSize, textColor, containerStyle, Boolify(isSubtle), wrap, maxWidth, textWeight, xamlTextBlock, hostConfig);
     }
+    
+    HRESULT AddTextInlines(
```

Comment:
>AddTextInlines [](start = 12, length = 14)

This turned out to be a lot of code, so I'm going to move it all into a helper file. Coming in a future commit (so the diffs are easier to see on this one).